### DISCUSSION TITLE
How do I reference a packaged field on an standard object

### DISCUSSION THREAD

**Discussion by sue21123** [deejay-hub/timeline-lwc#303] [Q&A] [answered]
Hi,

I'm trying to set up a configuration for Contacts and Opportunities.  We have a managed package that uses a custom contact field:

Field Label: Constituent 
Field Name: Contact 
Namespace Prefix:  ucinn_ascendv2
API name: ucinn_ascendv2__Contact__c
Child Relationship name:  Opportunities_Contact

In the configuration, I'm using Contact as the Parent Object and Opportunity as the Object Name.  I'm not sure of the syntax to reference this field in the Relationship Name field.  I tried Opportunities_Contact but that did not work.

Thanks so much for your help,

Sue

**Comment by deejay-hub** [comment, marked as answer]
Check out [this](https://github.com/deejay-hub/timeline-lwc/discussions/166) discussion. Where we walk through a similar request for a different package.

In your case the Relationship Name field value should be : ucinn_ascendv2__Opportunities_Contact__r. Note the double underscore after the namespace and the double underscore at the end.

**Reply by sue21123** [reply]
Thank you so much, that worked!  I appreciate your quick reply.  
My end users love this app!
Sue
